src/PrestaShopBundle/Translation/Provider/FrontOfficeProvider.php

```diff
@@ -83,6 +83,15 @@ public function getDefaultCatalogue($empty = true)
         return $defaultCatalogue;
     }
 
+    /**
+     * @param null $theme
+     * @return MessageCatalogue
+     */
+    public function getDatabaseCatalogue($theme = null)
+    {
+        return parent::getDatabaseCatalogue('classic');
```

Comment:
I don't need an instance of Theme but I will replace it with `_PS_DEFAULT_THEME_NAME_`